### PR TITLE
Add support of update time for Cassandra, Mysql and Postgres

### PR DESCRIPTION
**Pull request by neil-xie**
**What changed?**
Add support of update time field for Cassandra, MySQL and Postgres.

**Why?**

**How did you test it?**
Test the changes locally.

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Comment by davidporter-id-au** [comment]
@neil-xie  What will happen if we deploy this change before the schema has been updated? Have we tested this?

**Comment by neil-xie** [comment]
> @neil-xie What will happen if we deploy this change before the schema has been updated? Have we tested this?

I only found this [link for updating Cassandra schema](https://engwiki.uberinternal.com/display/TE0CADENCE/Cassandra+Schema+Update+in+Monorepo), I don't know how we can update Mysql and Postgres. I tested these locally. We have a minimum compatible version check, so we need to update the schema when deploy the new version.

**Comment by neil-xie** [comment]
> 



> @neil-xie What will happen if we deploy this change before the schema has been updated? Have we tested this?

We have a minimum compatible version check for Cassandra and sql databases, so it will complain about the db version and throw error. Are we supposed to be backward compatible?

**Review by davidporter-id-au — APPROVED** [review]

File: common/persistence/dataVisibilityManagerInterfaces.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ type (
 		TaskList           string
 		IsCron             bool
 		NumClusters        int16
+		UpdateTimestamp    int64
```
[the remark this answers is not in the document]
Comment (neil-xie) [reply common/persistence/dataVisibilityManagerInterfaces.go]: Yeah, it is unix nano. I added comment to specify the unit

**Pull request merged**